Mathlib/Analysis/Convex/Between.lean

```diff
@@ -810,6 +810,18 @@ theorem wbtw_iff_sameRay_vsub {x y z : P} : Wbtw R x y z ↔ SameRay R (y -ᵥ x
     field_simp [(add_pos hr₁ hr₂).ne', hr₂.ne']
     ring
 
+theorem AffineIndependent.not_wbtw {a b c : P} (hT : AffineIndependent R ![a, b, c]) :
+    ¬ Wbtw R a b c := by
```

Comment:
I do now see @dwrensha's original argument for dropping this, since of course there are 5 other versions of this statement which we've skipped. I'll leave the decision up to @jsm28, who was thought about this type of thing before in nearby API.

Reply:
Yeah, it seems to me that most uses of `AffineIndependent.not_wbtw` will need to need to invoke `AffineIndependent.comp_embedding` to reorder indices, at which point it would have been easier to just use `AffineIndependent.not_wbtw_of_injective` in the first place. I am inclined to re-combine these two theorems into just `AffineIndependent.not_wbtw_of_injective`.

Reply:
I concur that `not_wbtw_of_injective` is the main form (and probably more usable than this one in common cases where the affine independence isn't given for a family definitionally of the form `![a, b, c]` in that order).

Reply:
Removed the not_wbtw theorem in 7c345214265895d447f4c27b77ef1323f354bb25. If we discover that form to be useful in the future, then we can always add it back later.